File: Easy/Java/448. Find All Numbers Disappeared in an Array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        List<Integer> missing = new ArrayList<>();
        int[] count = new int[nums.length];

        for(int i = 0; i < nums.length; i++)
            count[nums[i] - 1]++;
        for(int i = 0; i < nums.length; i++){
            if(count[i] == 0)
                missing.add(i + 1);
        }

        return missing;
    }
}

File: Easy/Python/448. Find All Numbers Disappeared in an Array.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        missing = []
        count = [0] * len(nums)

        for i in range(len(nums)):
            count[nums[i] - 1] += 1
        for i in range(len(nums)):
            if count[i] == 0:
                missing.append(i + 1)
        
        return missing
